db/deploy/functions/receive_applications.sql

```diff
@@ -0,0 +1,49 @@
+-- Deploy ccbc:functions/receive_applications to pg
+
+BEGIN;
+
+create or replace function ccbc_public.receive_applications() returns table(result_id int)  as
+$function$
+	declare
+        current_app    ccbc_public.application%ROWTYPE;
+        last_status    text;
+        cnt             int;
+	    applications refcursor; 
+
+    begin
+    select count(*) into cnt from ccbc_public.application
+		    where intake_id in (
+                select id from ccbc_public.intake where now() >= close_timestamp);
+
+    open applications for select id, intake_id
+		    from ccbc_public.application
+		    where intake_id in (
+                select id from ccbc_public.intake where now() >= close_timestamp);
+
+    loop
+        fetch applications into current_app;
+        exit when not found;
+
+        select s.status into last_status from ccbc_public.application_status s inner join
```

Comment:
We can probably use the `application_status` computed column function here to simplify this, we should add execute permission for that function. There's also the possibility that how we store an application status changes and then we'd have two places to have to check/change.

Reply:
@VB-at-Bis thoughts on this?